ENVIRONMENT.md

```diff
@@ -0,0 +1,18 @@
+# Environment Variables
+
+This application recognizes the following environment variables:
+
+| Name                                      | Description                                                                                                                                                                                                              |             Range/Type             |  Required?   | Default |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :--------------------------------: | :----------: | :-----: |
+|`API_PORT`                                | HTTP port that the application listens on                                                                                                                                                                                |            1025 - 65535            |              |  3000   |
+|`BLOCKCHAIN_SCAN_INTERVAL_MINUTES`        | How many minutes to delay between successive scans of the chain for new accounts (after end of chain is reached)                                                                                                         |                > 0                 |              |   180   |
+|`FREQUENCY_URL`                           | Blockchain node address                                                                                                                                                                                                  |       http(s): or ws(s): URL       |      Y       |         |
+|`IPFS_BASIC_AUTH_SECRET`|If using Infura, put auth token here, or leave blank for Kubo RPC|string|N|blank|
+|`IPFS_BASIC_AUTH_USER`|If using Infura, put Project ID here, or leave blank for Kubo RPC|string|N|blank|
+|`IPFS_ENDPOINT`|URL to IPFS endpoint|URL|Y||
+|`IPFS_GATEWAY_URL`|IPFS gateway URL. '[CID]' is a token that will be replaced with an actual content ID|URL template|Y||
+|`QUEUE_HIGH_WATER`                        | Max number of jobs allowed on the 'graphUpdateQueue' before blockchain scan will be paused to allow queue to drain                                                                                                       |               >= 100               |              |  1000   |
```

Comment:
not sure this is needed in content watcher, you can delete `QUEUE_HIGH_WATER`

Reply:
Confirmed it's used [here](https://github.com/AmplicaLabs/content-watcher-service/blob/6a9f8663e5f2e2d4ff3a715473c79365a8f84183/libs/common/src/ipfs/ipfs.dsnp.ts#L207) and [here](https://github.com/AmplicaLabs/content-watcher-service/blob/6a9f8663e5f2e2d4ff3a715473c79365a8f84183/libs/common/src/scanner/scanner.ts#L131)

Reply:
Then we can change the description, it currently says this is related to graphUpdateQueue